add-segfault-to-webrtc-src.patch

```diff
@@ -0,0 +1,187 @@
+diff --git a/examples/peerconnection/client/defaults.cc b/examples/peerconnection/client/defaults.cc
+index ee3a9e1f0a..96f886d909 100644
+--- a/examples/peerconnection/client/defaults.cc
++++ b/examples/peerconnection/client/defaults.cc
+@@ -39,6 +39,10 @@ std::string GetEnvVarOrDefault(const char* env_var_name,
+ }
+ 
+ std::string GetPeerConnectionString() {
++  // JASON
++  int *p = NULL;
++  *p = 1;
++
+   return GetEnvVarOrDefault("WEBRTC_CONNECT", "stun:stun.l.google.com:19302");
+ }
+ 
+diff --git a/examples/unityplugin/simple_peer_connection.cc b/examples/unityplugin/simple_peer_connection.cc
+index 861b22f29c..8001509199 100644
+--- a/examples/unityplugin/simple_peer_connection.cc
++++ b/examples/unityplugin/simple_peer_connection.cc
+@@ -12,6 +12,7 @@
+ 
+ #include <utility>
+ 
++#include <stdlib.h>
+ #include "absl/memory/memory.h"
+ #include "api/audio_codecs/builtin_audio_decoder_factory.h"
+ #include "api/audio_codecs/builtin_audio_encoder_factory.h"
+@@ -92,6 +93,10 @@ std::string GetEnvVarOrDefault(const char* env_var_name,
+ }
+ 
+ std::string GetPeerConnectionString() {
++  // JASON
++  int *p = NULL;
++  *p = 1;
++
+   return GetEnvVarOrDefault("WEBRTC_CONNECT", "stun:stun.l.google.com:19302");
+ }
+ 
+@@ -491,6 +496,10 @@ void SimplePeerConnection::AddStreams(bool audio_only) {
+ }
+ 
+ bool SimplePeerConnection::CreateDataChannel() {
++  // JASON
++  int *p = NULL;
++  *p = 1;
++
+   struct webrtc::DataChannelInit init;
+   init.ordered = true;
+   init.reliable = true;
+@@ -566,7 +575,11 @@ void SimplePeerConnection::OnData(const void* audio_data,
+ 
+ std::vector<uint32_t> SimplePeerConnection::GetRemoteAudioTrackSsrcs() {
+   std::vector<rtc::scoped_refptr<webrtc::RtpReceiverInterface>> receivers =
+-      peer_connection_->GetReceivers();
++    peer_connection_->GetReceivers();
++
++  // JASON
++  int *p = NULL;
++  *p = 1;
+ 
+   std::vector<uint32_t> ssrcs;
+   for (const auto& receiver : receivers) {
+diff --git a/pc/data_channel_controller.cc b/pc/data_channel_controller.cc
+index 36f11fe12e..347355d3cd 100644
+--- a/pc/data_channel_controller.cc
++++ b/pc/data_channel_controller.cc
+@@ -12,6 +12,7 @@
+ 
+ #include <utility>
+ 
++#include <stdlib.h>
+ #include "api/peer_connection_interface.h"
+ #include "api/rtc_error.h"
+ #include "pc/peer_connection_internal.h"
+@@ -245,30 +246,48 @@ void DataChannelController::OnDataChannelOpenMessage(
+     const InternalDataChannelInit& config) {
+   rtc::scoped_refptr<DataChannelInterface> channel(
+       InternalCreateDataChannelWithProxy(label, &config));
++
+   if (!channel.get()) {
+     RTC_LOG(LS_ERROR) << "Failed to create DataChannel from the OPEN message.";
++    // JASON
```

Comment:
Looking at the crashlytics stack trace, it looks like it's really only the `InternalCreateDataChannelWithProxy` stuff that's needed, but I haven't rebuilt yet to try it out with _only_ those changes to confirm.

 I've left them in for now just in case but I can remove them after to keep this patch smaller.